abci/example/kvstore/kvstore.go

```diff
@@ -413,33 +413,33 @@ func isValidatorTx(tx []byte) bool {
 	return strings.HasPrefix(string(tx), ValidatorPrefix)
 }
 
-func parseValidatorTx(tx []byte) ([]byte, int64, error) {
+func parseValidatorTx(tx []byte) (string, []byte, int64, error) {
 	tx = tx[len(ValidatorPrefix):]
 
 	//  get the pubkey and power
-	pubKeyAndPower := strings.Split(string(tx), "!")
-	if len(pubKeyAndPower) != 2 {
-		return nil, 0, fmt.Errorf("expected 'pubkey!power'. Got %v", pubKeyAndPower)
+	typeKeyAndPower := strings.Split(string(tx), "!")
```

Comment:
Can we change to `keyTypeAndPower` ?

Reply:
SGTM. CC'ing @sergio-mena for confirmation since he made the change.

Reply:
This has been ported to our `main` branch and backported to `v0.38.x`: cometbft/cometbft#1966.
If you update your base branch to point to our last release (`v0.38.5`) you will get rid of these changes in this PR

Reply:
Sure although I believe @marcello33 's point was the naming of the param from `typeKeyAndPower` to `keyTypeAndPower`.

Reply:
Makes sense, I'll open a tiny PR on CometBFT with this change

Reply:
Actually, the three elements of the string are `keyType`, `pubKey`, and `power`, in this order, so wouldn't it make more sense to have `typePubKeyPower` as name?

Reply:
SGTM, up to you really. You can also consider `pubKeyTypeAndPower`/`keyTypeAndPower `.

Reply:
Resolved